stdlib/char.gr

```diff
@@ -87,16 +114,22 @@ let rec code = (c: Char) => {
     }
   }
 
-  Memory.decRef(c)
+  Memory.decRef(char)
   Memory.decRef(WasmI32.fromGrain(code))
   tagSimpleNumber(result)
 }
 
-// Returns the Char for the given code point. Fails if the code point is invalid.
-// @param codePoint: Number - the Unicode code point
-// @returns Char
+/**
+ * Creates a character from the given Unicode scalar value.
+ * Throws an exception if the Unicode scalar value is invalid.
+ * 
+ * @param codePoint: The Unicode scalar value
```

Comment:
```suggestion
 * @param usv: The Unicode scalar value
```
Shall we rename this `usv` for "Unicode scalar value"? Also need to update the parameter and code.